.githooks/pre-commit

```diff
@@ -30,5 +30,13 @@ for FILE in ${STAGED_SH_FILES}; do
 	git add "${FILE}"
 done
 
+echo "==> Formatting changed JSON files..."
+STAGED_JSON_FILES=$(git diff --cached --name-only | grep ".json$")
+for FILE in ${STAGED_JSON_FILES}; do
+  prettyFile=$(jq . "${FILE}")
```

Comment:
[nit] In the future, we should consider moving away from a jq dep to something we can guarantee is in the system path.

Slight concern our code is brittle with external dependences that the pre-commit hook can't ensure are present.

Reply:
Right now we are installing the devtools used in the hook [here](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/e1be514ee2cf1bd444f236479d504c4bc65c5810/Makefile#L23). Updated the make file to install `jq` using brew

Reply:
reverted my changes as using brew makes it hard to install devtools in github action. I will create a ticket to migrate away from jq

Reply:
That then adds a dep on brew, which may not be on the build env. Don't think that's better :)

Reply:
https://jira.mongodb.org/browse/INTMDB-615